### PR TITLE
new recipe: nf-test

### PR DESCRIPTION
**Pull request by matthdsm**
New recipe for `nf-test`
https://code.askimed.com/nf-test/installation/

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by matthdsm** [comment]
@BiocondaBot please add label

**Comment by rpetit3** [comment]
@biocondabot please fetch artifacts

File: recipes/nf-test/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+BINARY_HOME=$PREFIX/bin
+PACKAGE_HOME=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+
+mkdir -p $PREFIX/bin
+mkdir -p $PACKAGE_HOME
+
+sed -i "s|APP_HOME=.*|APP_HOME=\"${PACKAGE_HOME}\"|g" nf-test
+
+cp nf-test* $PACKAGE_HOME
```
Comment (daler) [inline recipes/nf-test/build.sh:11]: @matthdsm looks like there are multiple files copied over; are there any other executables that need to be symlinked as well?

Comment (matthdsm) [reply recipes/nf-test/build.sh:11]: nope, there's only a jar file and the execution script!

**Comment by matthdsm** [comment]
Can I get an approve on this so I can merge?

File: recipes/nf-test/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    - openjdk >=11,<=18
+    - coreutils
+    - curl
+
+test:
+  commands:
+    - nf-test version
+
+about:
+  home: 'https://code.askimed.com/nf-test/'
```
Comment (ewels) [inline recipes/nf-test/meta.yaml:31]: I think GitHub URLs could be here for source and issues and stuff off the top of my head? But it's minor.

**Review by ewels — APPROVED** [review]
LGTM 🚀

**Comment by ewels** [comment]
@BiocondaBot please merge

**Pull request merged**